packages/pipeline/src/parsers/radar_orders/index.ts

```diff
@@ -0,0 +1,91 @@
+import { BigNumber } from '@0x/utils';
+import { Web3Wrapper } from '@0x/web3-wrapper';
+import { RadarBook, RadarMarket, RadarOrderType, RadarSignedOrder } from '@radarrelay/types';
+import * as R from 'ramda';
+
+import { aggregateOrders, GenericRawOrder } from '../utils';
+
+import { TokenOrderbookSnapshot as TokenOrder } from '../../entities';
+import { OrderType } from '../../types';
+
+/**
+ * Marque function of this file.
+ * 1) Takes in orders from an orderbook,
+ * other information attached.
+ * @param radarOrderbook A raw orderbook that we pull from the radar API.
+ * @param radarMarket An object containing market data also directly from the API.
+ * @param observedTimestamp Time at which the orders for the market were pulled.
+ * @param source The exchange where these orders are placed. In this case 'radar'.
+ */
+export function parseRadarOrders(
+    radarOrderbook: RadarBook,
+    radarMarket: RadarMarket,
+    observedTimestamp: number,
+    source: string,
```

Comment:
radar results would always have `source = radar` right? We could define RADAR_SOURCE here instead so the other two files don't need to handle any concept of source.

Reply:
This is how every other parser does it, but change change those too

Reply:
Actually looking into it again, I think I prefer the current format.

Reply:
I must agree with Xianny that this feels weird to always pass `radar` for `source` when the function isn't used for anything else besides Radar.